runners/flink/src/test/java/org/apache/beam/runners/flink/streaming/UnboundedSourceWrapperTest.java

```diff
@@ -68,7 +68,6 @@
 import org.slf4j.LoggerFactory;
 
 /** Tests for {@link UnboundedSourceWrapper}. */
-@RunWith(Enclosed.class)
```

Comment:
Won't this disable all tests of the nested classes? Was this intended?

Reply:
Ah, I could have left this in place. This was a line from my earlier PR. Before that time, this was never there. It turns out that many test runners will find the inner classes anyhow. But I do think this should be restored.